diff --git a/projects/examples/source/13.21.cpp b/projects/examples/source/13.21.cpp
index c2175392..7a138d98 100644
--- a/projects/examples/source/13.21.cpp
+++ b/projects/examples/source/13.21.cpp
@@ -4,8 +4,8 @@
 
 //////////////////////////////////////////////////////////////////////////////////
 
-#include <zipper/3.0.2/Zipper/Unzipper.hpp>
-#include <zipper/3.0.2/Zipper/Zipper.hpp>
+#include <Zipper/Unzipper.hpp>
+#include <Zipper/Zipper.hpp>
 
 //////////////////////////////////////////////////////////////////////////////////
 
